airflow/www/templates/airflow/gantt.html

```diff
@@ -65,7 +65,7 @@
       .taskStatus(data.taskStatus)
       .height(data.height)
       .selector('.gantt')
-      .tickFormat("%H:%M:%S");
+      .tickFormat('%H:%M:%S');
```

Comment:
```suggestion
      .tickFormat("%H:%M:%S");
```

Where possible we like to avoid "un-necessary" changes to files that are otherwise not touched because they make it harder to back-port changes.